UnicodeDecodeError when testing with python 3.5.2

Trying to update the python-nbformat package to 4.2.0 in Fedora but I'm getting the following test error:
```
+ py.test-3.5 -v nbformat/tests/
============================= test session starts ==============================
platform linux -- Python 3.5.2, pytest-3.0.5, py-1.4.31, pluggy-0.4.0 -- /usr/bin/python3
cachedir: .cache
rootdir: /builddir/build/BUILD/nbformat-4.2.0, inifile:
collecting ... collected 16 items

nbformat/tests/test_api.py::TestAPI::test_read PASSED
nbformat/tests/test_api.py::TestAPI::test_read_write_path PASSED
nbformat/tests/test_api.py::TestAPI::test_write_downgrade_2 PASSED
nbformat/tests/test_convert.py::TestConvert::test_downgrade_3_2 PASSED
nbformat/tests/test_convert.py::TestConvert::test_open_current PASSED
nbformat/tests/test_convert.py::TestConvert::test_upgrade_2_3 PASSED
nbformat/tests/test_convert.py::TestConvert::test_upgrade_downgrade_4_3_4 PASSED
nbformat/tests/test_reader.py::TestReader::test_read PASSED
nbformat/tests/test_validator.py::TestValidator::test_future PASSED
nbformat/tests/test_validator.py::TestValidator::test_invalid PASSED
nbformat/tests/test_validator.py::TestValidator::test_nb2 PASSED
nbformat/tests/test_validator.py::TestValidator::test_nb3 PASSED
nbformat/tests/test_validator.py::TestValidator::test_nb4 PASSED
nbformat/tests/test_validator.py::TestValidator::test_nb4_document_info PASSED
nbformat/tests/test_validator.py::TestValidator::test_nb4custom FAILED
nbformat/tests/test_validator.py::TestValidator::test_validation_error PASSED

=================================== FAILURES ===================================
_________________________ TestValidator.test_nb4custom _________________________

self = <nbformat.tests.test_validator.TestValidator testMethod=test_nb4custom>

    def test_nb4custom(self):
        """Test that a notebook with a custom JSON mimetype passes validation"""
        with self.fopen(u'test4custom.ipynb', u'r') as f:
>           nb = read(f, as_version=4)

nbformat/tests/test_validator.py:47:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
nbformat/__init__.py:141: in read
    return reads(fp.read(), as_version, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <encodings.ascii.IncrementalDecoder object at 0x7fc2bdbbe748>
input = b'{\n "cells": [\n  {\n   "cell_type": "code",\n   "execution_count": 2,\n   "metadata": {\n    "collapsed": false\n  ...\n   },\n   "outputs": [],\n   "source": []\n  }\n ],\n "metadata": {\n },\n "nbformat": 4,\n "nbformat_minor": 0\n}\n'
final = True

    def decode(self, input, final=False):
>       return codecs.ascii_decode(input, self.errors)[0]
E       UnicodeDecodeError: 'ascii' codec can't decode byte 0xf0 in position 217: ordinal not in range(128)

/usr/lib64/python3.5/encodings/ascii.py:26: UnicodeDecodeError
===================== 1 failed, 15 passed in 0.58 seconds ======================
```

--- a/nbformat/tests/base.py
+++ b/nbformat/tests/base.py
@@ -7,13 +7,13 @@ Contains base test class for nbformat
 
 import os
 import unittest
-
+import io
 
 class TestsBase(unittest.TestCase):
     """Base tests class."""
 
-    def fopen(self, f, mode=u'r'):
-        return open(os.path.join(self._get_files_path(), f), mode)
+    def fopen(self, f, mode=u'r',encoding='utf-8'):
+        return io.open(os.path.join(self._get_files_path(), f), mode, encoding=encoding)
 
 
     def _get_files_path(self):

